CHANGELOG.md

```diff
@@ -1,10 +1,18 @@
+## 10.0.0
+### General
+- The `compressionQuality` property in the `pickFiles` method now defaults to `0`, and `allowCompression` is set to `false`. [@vicajilau](https://github.com/vicajilau).
```

Comment:
I think you removed the wrong duplicated section here? Now the breaking change label is gone.